transfers/transfer.py

```diff
@@ -301,16 +321,16 @@ def start_transfer(ss_url, ts_location_uuid, ts_path, depth, am_url, user_name,
     if not response.ok or resp_json.get('error'):
         LOGGER.error('Unable to start transfer.')
         LOGGER.error('Response: %s', resp_json)
-        new_transfer = Unit(path=target, unit_type='transfer', status='FAILED', current=False)
+        new_transfer = models.Unit(path=target, unit_type='transfer', status='FAILED', current=False)
         session.add(new_transfer)
         return None
 
     # Run all scripts in pre-transfer directory
     # TODO what inputs do we want?
     run_scripts('pre-transfer',
-        resp_json['path'],  # Absolute path
-        'standard',  # Transfer type
-    )
+                resp_json['path'],  # Absolute path
+                'standard',  # Transfer type
+                )
```

Comment:
Nitpick about moving `'pre-transfer'` to the next line instead of re-indenting everything applies here too.